docker/Dockerfile

```diff
@@ -93,6 +93,9 @@ ARG RAILS_ENV=production
 ENV RAILS_ENV ${RAILS_ENV}
 ENV BUNDLE_PATH="/gems"
 
+ARG NODE_OPTIONS="--openssl-legacy-provider"
+ENV NODE_OPTIONS ${NODE_OPTIONS}
+
```

Comment:
@abehnamfard Can you please share what the error was and how does this fix it? The same line is available in line 16-17. As far as I see there is no JS bundling action taken post this line.

@vishnu-narayanan Please take a look.

Reply:
Lines 16 & 17 belong to the pre-build stage of the docker.
This change will fix the error on the webpack container when it is running (Build Stage).

Reply:
<img width="831" alt="Screenshot 2024-01-19 at 10 32 26 PM" src="https://github.com/chatwoot/chatwoot/assets/16982564/c50b42b6-6ef3-428a-b9d2-75da6466da04">